addons/im_livechat/static/src/models/mobile_messaging_navbar/mobile_messaging_navbar.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module **/
+
+import { patchRecordMethods } from '@mail/model/model_core';
+// ensure that the model definition is loaded before the patch
+import '@mail/models/mobile_messaging_navbar_view/mobile_messaging_navbar_view';
+
+patchRecordMethods('MobileMessagingNavbarView', {
+    /**
+     * @override
+     */
+    _computeTabs() {
+        if (this.discuss) {
+            return [...this._super(), {
+                icon: 'fa fa-comments',
+                id: 'livechat',
+                label: this.env._t("Livechat"),
+            }];
```

Comment:
functional remark: on desktop the livechat sidebar category doesn't show if there are no livechat on it.

We should probably do the same here. No need to show that when useless, especially to non-livechat operators.